solutions/src/test/java/ru/sberbank/school/task13/BeanFieldCopierImplTest.java

```diff
@@ -0,0 +1,72 @@
+package ru.sberbank.school.task13;
+
+import lombok.*;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+class BeanFieldCopierImplTest {
+    private static BeanFieldCopier copier;
+    private static User bob;
+    private static Cat cat;
+
+    @BeforeAll
+    static void init() {
+        copier = new BeanFieldCopierImpl();
+        bob = new User("Bob", 5, 456-34-34, "bob@mail.com");
```

Comment:
WhitespaceAround: &#39;-&#39; is not preceded with whitespace.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-1026067987>*